README.md

```diff
@@ -194,6 +194,23 @@ following rules:
 For more details about authorization in LoopBack 4, refer to
 https://loopback.io/doc/en/lb4/Loopback-component-authorization.html.
 
+### Tests
```

Comment:
I see you have this under the Authentication section, but can you move it under the usage section?

Reply:
Updated